Weather App/WeathersList/controller/WeatherDetailsViewController.swift

```diff
@@ -1,23 +1,62 @@
 import Foundation
+import SnapKit
 import UIKit
 
 class WeatherDetailsViewController: UIViewController {
     @IBOutlet var cityNameLabel: UILabel!
     @IBOutlet var temperatureLabel: UILabel!
+    @IBOutlet var sunriseLabel: UILabel!
+    @IBOutlet var sunsetLabel: UILabel!
+    @IBOutlet var humadityLabel: UILabel!
     @IBOutlet var pressureLabel: UILabel!
-    @IBOutlet var humidityLabel: UILabel!
-    @IBOutlet var weatherDescriptionLabel: UILabel!
-    var weather: CityWeather?
+    @IBOutlet var windSpeedLabel: UILabel!
+    @IBOutlet var cloudinessLabel: UILabel!
+    @IBOutlet var weatherImageView: UIImageView!
+    @IBOutlet var forecastHoursCollection: UICollectionView!
+    @IBOutlet var daysForecastsCollection: UICollectionView!
+    private lazy var hoursForecastDataSource = HoursForecastDataSource(weather: weather)
+    private lazy var daysForecastDataSource = DaysForecastDataSource(weather: weather)
+
+    var weather: CityWeather!
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        guard let weather = weather else {
-            return
+        guard weather != nil else {
+            fatalError("Bad weather init")
         }
+        configureView()
+    }
+
+    private func configureView() {
         cityNameLabel.text = weather.name
         temperatureLabel.text = "\(Int(weather.mesurements.temperature)) °C"
-        pressureLabel.text = "\(weather.mesurements.pressure) hPa"
-        humidityLabel.text = "\(weather.mesurements.humidity) %"
-        weatherDescriptionLabel.text = "\(weather.description[0].descriptionWeather)"
+        humadityLabel.text = "\(weather.mesurements.humidity)%"
+        pressureLabel.text = "\(weather.mesurements.pressure)hPa"
+        windSpeedLabel.text = "\(weather.wind.velocity)km/h"
```

Comment:
I think you can use formatters for all measurements, I know, you can use UnitPressure https://developer.apple.com/documentation/foundation/unitpressure

Reply:
👍